_posts/2020-03-15-learning-to-share-accomplishments-1.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Learning to Share Accomplishments: Part 1 - How to Share Accomplishments"
+date: 2020-03-15T09:00:00-04:00
+permalink: /2020/03/15/learning-to-share-accomplishments-1/
+author_profile: true
+toc: true
+toc_label: "Post Contents"
+toc_icon: "file-alt"
+medium_post:
+  - ""
+categories:
+  - Accomplishments
+  - Job Hunting
+tags:
+  - 
+---
+
+This post is a part of a series on job hunting and job interviewing.
+{: .notice--info}
+
+<figure>
+    <img src='https://www.hdwallpaper.nu/wp-content/uploads/2017/04/cat-11.jpg' alt='Until I figure out a better image' width="50%" height="50%" />
+    <figcaption>Until I figure out a better image to put here.....</figcaption>
+</figure>
+
+In [my last post](/2019/09/02/reflections-on-the-latest-round-of-job-hunting/), I talked about my last round of job 
+hunting. More specifically, I mention one of the pieces of feedback was on how I come off as a bit more of an 
+"individual contributor" (someone who just takes tasks and does them) as opposed to someone who actually designs or 
```

Comment:
I actually thought of "individual contributor" as someone who has only worked alone / not on a team for a long time because that's my background / venerability so I'm glad it was defined here. <3

Reply:
Yeah, I thought that at one point too. It's just a single contributor to a project, not necessarily a solo worker. Perhaps not the best naming strategy...